src/main/java/ai/lucidtech/las/sdk/Client.java

```diff
@@ -59,11 +55,25 @@ public Client(String endpoint, Credentials credentials) {
      * @param consentId An identifier to mark the owner of the document handle
      * @return Response from API
      */
-    public JSONObject postDocuments(ContentType contentType, String consentId) throws IOException {
+    public JSONObject createDocument(
```

Comment:
We should allow not needing to pass options if it's empty. I suggest method overloading unless Java has support for optionals